File: Problem 1/p1v1.cpp
#include <ostream>
#include <iostream>
int main(void)
{
	int sum = 0;
	for (int i = 0; i < 1000; i++) {
		if (i % 3 == 0) { sum += i; }
		else if (i % 5 == 0) { sum += i; }
	}
	std::cout << sum << std::endl;
	return 0;
}

File: Problem 1/p1v1.hs
main = do
    let sum = 0
    for [1..10] $ \i -> do
        if i `mod` 3 == 0
            then let sum = sum + i
            else if i `mod` 5 == 0
                    then let sum = sum + i
                    else return ()
    print sum